### ISSUE TITLE
AuthorizationConfig property on ecs.EFSVolumeConfiguation class needs required value set - not part of release 2.6.3

### ISSUE THREAD

**Issue by svsShiva** [cloudtools/troposphere#1831]
Although this issue (https://github.com/cloudtools/troposphere/issues/1806) is resolved with this PR (https://github.com/cloudtools/troposphere/pull/1808), I am still facing the same issue. 

Seems like release 2.6.3 (https://pypi.org/project/troposphere/) does not include the fix.

Correct me if I am wrong - Thanks

**Comment by PatMyron** [comment]
Not released to PyPI yet, but it is possible to `pip install` directly from git repos if you need an immediate workaround:
```bash
pip install git+https://github.com/cloudtools/troposphere --upgrade
```

**Comment by svsShiva** [comment]
Thanks for the response. However, after installing it from GitHub source, I am seeing below issues.
 
E0401: Unable to import 'awacs.sts' (import-error)
E0401: Unable to import 'awacs.aws' (import-error)
 
Can anyone help me out with this?

**Comment by markpeek** [comment]
This was included in the 2.6.4 release.
